internal/api/grpc_definitions_service.go

```diff
@@ -176,7 +182,12 @@ func (s *GrpcDefinitionsService) prepareIntegrationResponse(integration coremode
 			ICE:  int32(integration.AutoPiPowertrainToTemplateID[coremodels.ICE]),
 			PHEV: int32(integration.AutoPiPowertrainToTemplateID[coremodels.PHEV]),
 		},
-	}, nil
+		Points: int64(integration.Points),
+	}
+	if integration.ManufacturerTokenID != 0 {
+		integ.ManufacturerTokenId = uint64(integration.TokenID)
+	}
```

Comment:
Same question here. Actually, is this a bug? We're assigning the integration's token id the "manufacturer token id" field?